tests/scripts/build-release-downstream.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+set -ex
+
+MAKE='make --debug=v --output-sync'
+
+# build and push rook image
+$MAKE build BUILD_REGISTRY=local
+build_Image="local/ceph-amd64:latest"
+git_hash=$(git rev-parse --short "${GITHUB_SHA}")
+tag_Image=quay.io/ocs-dev/rook-ceph:v${BRANCH_NAME}-$git_hash
```

Comment:
nit: Always use either `snake_case` or `camelCase` don't use a combination of both of them.

Reply:
Also, since we are using branch names here, the preceding `v` can be removed from the image tag.

Reply:
So env variables are in capitals, and other variables are in camelCase...
So to understand the difference between what is coming in CI Runtime...

Reply:
> Also, since we are using branch names here, the preceding v can be removed from the image tag.

v gives a  better idea that we are giving a versioning, which we follow at other places

Reply:
But removing `v` is convenient I can remove it

Reply:
> So env variables are in capitals, and other variables are in camelCase... So to understand the difference between what is coming in CI Runtime...

I don't see any variable in camelCase they are a combo of snake and camelCase

Reply:
Got it,